eos-s3/btn_counter/Makefile

```diff
@@ -1,14 +1,12 @@
-mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
-current_dir := $(patsubst %/,%,$(dir $(mkfile_path)))
-TOP:=top
-VERILOG:=btn_counter.v
-DEVICE := ql-eos-s3
-PARTNAME := PD64
-PCF:=chandalar.pcf
-
 ifdef F4PGA_USE_DEPRECATED
 all:
-	ql_symbiflow -compile -d ${DEVICE} -P ${PARTNAME} -v ${VERILOG} -t ${TOP} -p ${PCF}
```

Comment:
I actually liked that the parameters were passed as variables. That way it was more readable - it was obvious what the options are for

Reply:
I updated it.